internal/pkg/aws/cloudformation/stackstatus.go

```diff
@@ -52,7 +52,8 @@ func (ss StackStatus) UpsertInProgress() bool {
 	return ss == cloudformation.StackStatusCreateInProgress || ss == cloudformation.StackStatusUpdateInProgress
 }
 
-func (ss StackStatus) Success() bool {
+// IsSuccessful returns true if the resource mutated successfully.
+func (ss StackStatus) IsSuccessful() bool {
```

Comment:
Why do we use `IsSuccessful` instead of just `Successful` 🤔

Reply:
No reason, I just picked one - I think both are pretty readable 🤷 renamed the method to `IsSuccess` to keep it a bit shorter